src/inmanta/protocol/rest/__init__.py

```diff
@@ -645,3 +693,10 @@ async def _execute_call(
         except Exception as e:
             LOGGER.exception("An exception occurred during the request.")
             raise exceptions.ServerError(str(e.args))
+
+    @abc.abstractmethod
+    async def get_policy_engine_slice(self) -> Optional["PolicyEngineSlice"]:
+        """
+        Return the policy-engine slice or None if no such slice exists because we are not running on the server.
+        """
+        raise NotImplementedError()
```

Comment:
I would expect this to only exist on the server side?

Reply:
I am not sure what you mean. Could you make a concrete proposal of what you would like to be changed?